functions/index.js

```diff
@@ -48,10 +48,10 @@ exports.dailycc = functions.https.onRequest(async (req, res) => {
 exports.affirmations = functions.https.onRequest((req, res) => {
 	discordClient.on('ready', async () => {
 		try {
-			const quote = await postAffirmation();
-			if (quote) {
-				discordClient.channels.cache.get(testChannel).send(quote);
-				res.send(quote);
+			const message = await postAffirmation();
+			if (message) {
+				discordClient.channels.cache.get(affirmationsChannel).send(message);
+				res.send(message);
 			} else res.send('Cannot retrieve data');
```

Comment:
Could you place this into another brace. Everything else looks good

Reply:
Write a [guard clause](https://learningactors.com/javascript-guard-clauses-how-you-can-refactor-conditional-logic/). So if(!message) return res.send('Cannot retrieve data') then write the normal functionality after for better readability. This is a way to avoid conditional statements